### PR TITLE
Gh 866 confirm pk screen

### PR DESCRIPTION
**Pull request by sche**

### CODE REVIEW

File: Multisig/Assets.xcassets/ico-confirm-thin.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
```
Comment (MouazAlzahabi) [inline Multisig/Assets.xcassets/ico-confirm-thin.imageset/Contents.json:2]: This one is exactly same as checkmark.circle right?

File: Multisig/UI/Settings/ImportOwnerKey/PrivateKeyController.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+//  PrivateKeyController.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 25.01.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class PrivateKeyController {
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/ImportOwnerKey/PrivateKeyController.swift:11]: Do we need to have this separated in controller? 
Isn't app settings enough?

If we want to go with this, how about moving remove key here also ?

File: Multisig/Assets.xcassets/ico-confirm-thin.imageset/Contents.json (hunk 1 above)
Comment (sche) [reply Multisig/Assets.xcassets/ico-confirm-thin.imageset/Contents.json:2]: not exactly. It is thinner and has small margins 🤷‍♂️

File: Multisig/UI/Settings/ImportOwnerKey/PrivateKeyController.swift (hunk 2 above)
Comment (sche) [reply Multisig/UI/Settings/ImportOwnerKey/PrivateKeyController.swift:11]: Yes, placing removing key logic is a good idea

File: Multisig/Assets.xcassets/ico-confirm-thin.imageset/Contents.json (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/Assets.xcassets/ico-confirm-thin.imageset/Contents.json:2]: Ah I see, If you want let's make sure if we can use the same icon

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**